lambda-pages/src/App.js

```diff
@@ -0,0 +1,62 @@
+// Dependencies
+import React, { Component } from 'react';
+import { Route, withRouter } from 'react-router-dom';
+import fuzzyFilterFactory from 'react-fuzzy-filter';
+// Components
+import CreateNoteView from './views/CreateNoteView';
+import EditNoteView from './views/EditNoteView';
+import LoginView from './views/LoginView';
+import NoteView from './views/NoteView';
+import NotesView from './views/NotesView';
+import Sidebar from './components/Sidebar';
+// Styles
+import './styles/App.css';
+
+const { InputFilter, FilterResults } = fuzzyFilterFactory();
+
+class App extends Component {
+  state = {
+    path: this.props.location.pathname
```

Comment:
I'm a little confused as to why you're putting this on state, since you have the information on props already? It means you need to have the componentDidUpdate method. But couldn't you just reference this.props.location.pathname wherever you currently reference this.state.path?

If the problem is that it's too long, you could just have a method like this:
`getPath = () => this.props.location.pathname`

Reply:
or you could just destructure the pathname as its own variable in your render function...